app/models/cms/section.rb

```diff
@@ -1,4 +1,5 @@
 class CMS::Section < ApplicationRecord
+  include CMS::SystemName
```

Comment:
I'm afraid that adding the validation of `system_name` from the module will invalidate many sections existing in the DB, there are already too many in a "free form" :(